db-init/src/main/resources/database/migrations/V4.3__Update_Claim_Table.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE claim 
+ADD COLUMN vbms_id VARCHAR,
+ADD COLUMN off_ramp_reason VARCHAR, 
```

Comment:
Should be in claim_submission. See https://dsva.slack.com/archives/C01Q7979Z7D/p1675959892218709?thread_ts=1675912916.651659&cid=C01Q7979Z7D

Reply:
Done in V4.7 and V4.8 migrations.